webapi/src/main/scala/org/knora/webapi/util/DateUtilV1.scala

```diff
@@ -96,7 +96,24 @@ object DateUtilV1 {
         val daysInMonth = Calendar.DAY_OF_MONTH // will be used to determine the number of days in the given month
         // val monthsInYear = Calendar.MONTH // will be used to determine the number of months in the given year (generic for other calendars)
 
-        val dateSegments = dateString.split(InputValidation.PrecisionSeparator)
+        var era: Option[Int] = None
+
+        var date: String =dateString
+        if (dateString.contains(" ")) {
```

Comment:
If there is no era indicated, just set it to `AD` so the subsequent handling will be no different.